receipt_dynamo/receipt_dynamo/entities/compaction_run.py

```
@@ -0,0 +1,273 @@
+    lines_started_at: Optional[str | datetime] = None
+    lines_finished_at: Optional[str | datetime] = None
+    words_started_at: Optional[str | datetime] = None
+    words_finished_at: Optional[str | datetime] = None
+
+    lines_error: str = ""
+    words_error: str = ""
+
+    lines_merged_vectors: int = 0
+    words_merged_vectors: int = 0
+
+    created_at: str | datetime = datetime.utcnow()
```

Ruff (0.13.1)

Do not perform function call `datetime.utcnow` in dataclass defaults

(RUF009)